ansible/group_vars/all/defaults.yaml

```diff
@@ -45,3 +45,6 @@ kubernetes_bins:
   - kubelet
 
 sysprep: true
+offline_local_bundle_path: ""
+offline_remote_bundle_dir: "/root/.konvoy/offline"
+offline_remote_bundle_file: "offline.tar.gz"
```

Comment:
Is this used anywhere? I suspect the design changed, and this is now dead code.

Reply:
It is used, see my next comment